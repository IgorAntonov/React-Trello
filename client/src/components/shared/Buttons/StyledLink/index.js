import styled from 'styled-components';
import PropTypes from 'prop-types';
import { Link } from 'react-router-dom';

import { styles } from '../commonStyles';

export const StyledLink = styled(Link)`
  ${styles}
`;

Link.propTypes = {
  to: PropTypes.string.isRequired
};
