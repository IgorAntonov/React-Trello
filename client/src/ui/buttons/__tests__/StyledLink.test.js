import React from 'react';
import { shallow } from 'enzyme';

import { StyledLink } from '../StyledLink';

test('renders', () => {
  const wrapper = shallow(<StyledLink to="/" />);

  expect(wrapper.exists()).toBe(true);
});

