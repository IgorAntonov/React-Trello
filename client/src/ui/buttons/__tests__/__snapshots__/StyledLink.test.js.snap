// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StyledLink/> should match snapshot 1`] = `
<Link
  className="StyledLink-khUpca gFlSBf"
  replace={false}
  to="/"
/>
`;
